docs/toc.md

```diff
@@ -13,29 +13,34 @@ layout: page
 
 <link rel="stylesheet" href="assets/main.css">
 
-<font size="+4"><p style="text-align: center;">Dafny Documentation</p></font>
+<font size="+4"><p style="text-align: center;">The Dafny Language and Software Verification System</p></font>
```

Comment:
Same as above; should be a title, not a `<font>`.  This will help with accessibility, too.

Reply:
Done